x11-misc/libinput-gestures/libinput-gestures-2.45.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5..7} )
+
+inherit python-single-r1 xdg-utils
+
+DESCRIPTION="Actions gestures on your touchpad using libinput"
+HOMEPAGE="https://github.com/bulletmark/libinput-gestures"
+if [[ ${PV} == *9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/bulletmark/${PN}.git"
+else
+	SRC_URI="https://github.com/bulletmark/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+fi
+
+LICENSE="GPL-3+"
+SLOT="0"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+RDEPEND="${PYTHON_DEPS}
+	>=dev-libs/libinput-1.8.0
+	x11-misc/xdotool
+	x11-misc/wmctrl"
+DEPEND=">=dev-libs/libinput-1.8.0
```

Comment:
This has a subslot, and its in DEPEND so most likely it gets linked during compile. You should add a subslot binder by appending `:=` there. Rest looks good!

Reply:
libinput-gestures is a python script, it doesn't link with libinput

Reply:
Is `DEPEND` needed here at all, then?

Reply:
@juippis Yes, because another python script uses libinput during install (build) phase